packages/platform-android/src/commands/runAndroid/index.js

```diff
@@ -296,6 +293,103 @@ function startServerInNewWindow(
   );
 }
 
+function findPreviousTerm(content, endPos) {
```

Comment:
I think this is duplicate from a separate file. We should remove it entirely.

Reply:
Yea, these function helpers are very similar to each other with mostly single-line changes. Could you refactor them so we avoid the duplication?

Reply:
Looks like this was a rebase failure. I've updated the PR properly and now it's gone.